tests/cpptests/test_cpp_forkgc.cpp

```diff
@@ -82,17 +88,16 @@ class FGCTest : public ::testing::Test {
   void runGcThread() {
     fgc = reinterpret_cast<ForkGC *>(get_spec(ism)->gc->gcCtx);
```

Comment:
Can we move this line to the setup function? IMO it doesn't make much sense to have it here